tests/pillar/network.sls

```diff
@@ -24,3 +24,26 @@ linux:
         #type: vlan
         #use_interfaces:
         #- interface: ${linux:interface:eth0}
+    proxy:
+      enabled: true
+      system:
+        enabled: true
+        #https: https://127.0.2.1:443
+        #http: https://127.0.2.2:443
+        ftp: ftp://127.0.2.3:2121
+        noproxy:
+          - 192.168.0.1
+          - 192.168.0.2
+          - .local
+      apt:
+        enabled: true
+        https: http://127.3.3.1:4443
+        #http: http://127.3.3.2
+        ftp: none
+      noproxy:
```

Comment:
just keep the posibility to override on system/noproxy level the same as https/http/ftp etc. In real life  system/noproxy will not be used. Might come in place if we would differenciate between /etc/environment and content under /etc/profile.d.